src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,69 @@
+package command;
+
+import exception.LunaException;
+
+import java.time.LocalDate;
+
+import storage.Storage;
+
+// Import Tasks
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+
+import ui.Ui;
+
+/**
+ * Encapsulates a user instruction to create a new task.
+ *
+ * @author fannyjian
+ */
+public class AddCommand extends Command {
+    private Task task;
+
+    /**
+     * Initialises a command to create a new Todo.
+     *
+     * @param des Description of task.
+     */
+    public AddCommand(String des) {
```

Comment:
Perhaps you can consider a more understandable and intuitive variable name like "description"? instead of shortening it to "des"